src/components/WebexSettings/Select.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Select component
+ *
+ * @param {object} props Data passed to the component
+ * @param {object[]} props.options Array of HMTL option elements
```

Comment:
they are not (yet) html option elements, they are just simple objects with 2 props: value and label.

I see that you only use value, but value usually is some id (eg: camera device id) so we need a label to display to the user.

Please document these 2 props in the jsdoc, eg:
```js
* @param {string} props.options[].value  The value of the option
```